specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-03-01/examples/Caches_forceGsi.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "cacheName": "sc",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-03-01"
+  },
+  "responses": {
+    "204": {}
```

Comment:
This might require some additional discussion, but I think this should be 202 Accepted (and return an async op URL) to monitor the progress of this operation.